src/validators/mod.rs

```diff
@@ -553,19 +553,40 @@ macro_rules! validator_match {
     };
 }
 
+// Used when creating the base validator instance, to avoid reusing the instance
+// when unpickling:
+pub fn build_validator_base(
+    schema: &Bound<'_, PyAny>,
+    config: Option<&Bound<'_, PyDict>>,
+    definitions: &mut DefinitionsBuilder<CombinedValidator>,
+) -> PyResult<CombinedValidator> {
+    _build_validator(schema, config, definitions, false)
+}
+
 pub fn build_validator(
```

Comment:
Maybe this should similarly be `CombinedValidator::build`?

Reply:
I can create a separate issue but I have limited bandwidth currently to work on this